lib/fetchHttpClient.ts

```diff
@@ -34,7 +36,7 @@ export class FetchHttpClient implements HttpClient {
       statusCode: fetchResponse.status,
       headers: responseHeaders,
       textBody: async () => await fetchResponse.text(),
-      deserializedBody: async () => await fetchResponse.json()
+      serializedBody: async () => await fetchResponse.json()
```

Comment:
Note that [json parses the body content](https://developer.mozilla.org/en-US/docs/Web/API/Body/json) so I think the property name doesn't seem to fit with the implementation here.